tests/tier2/tc_2018_validate_server_option_in_etc_virtwho_d.py

```diff
@@ -31,7 +32,10 @@ def test_run(self):
         logger.info(">>>step2: server option is wrong value")
         self.vw_option_update_value(option_tested, "xxxxxx", config_file)
         data, tty_output, rhsm_output = self.vw_start()
-        msg_list = ["Name or service not known|Connection timed out|Failed to connect|Error in .* backend"]
+        msg_list = ["Name or service not known|"
+                    "Connection timed out|"
+                    "Failed to connect|"
+                    "Error in .* backend"]
```

Comment:
I have a idea is: could we use a dict or list to instead of msg_list, such as:
```
msg_list = {
'aaaa',
'bbbb',
'cccc'}
```
and then we can change it by msg_list.add or msg_list.delete.
Maybe we can improve it by another PR.